patches/third_party-blink-renderer-core-frame-reporting_observer.idl.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/third_party/blink/renderer/core/frame/reporting_observer.idl b/third_party/blink/renderer/core/frame/reporting_observer.idl
+index a248a08fcd1e3566c838f26393fd64c877bab11b..a1ae499dab51d79756e339cfd2ac508675f97b90 100644
+--- a/third_party/blink/renderer/core/frame/reporting_observer.idl
++++ b/third_party/blink/renderer/core/frame/reporting_observer.idl
+@@ -7,6 +7,7 @@
+ callback ReportingObserverCallback = void (sequence<Report> reports, ReportingObserver observer);
+ 
+ [
++    ContextEnabled=ReportingObservers,
```

Comment:
for reference what exactly does this do?

Reply:
@bridiver = With `ContextEnabled` we'll be able to `--enable-blink-features=ReportingObservers"` at runtime to enable `ReportingObservers`. Default behavior is disabled.